recipes/foundry-compute-modules/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "foundry-compute-modules" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/foundry_compute_modules-{{ version }}.tar.gz
+  sha256: 6812f7839b7ca1ee402bb26662f3b4d81ea2776c4b2efde559dfd31543c83fb3
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python 3.9
```

Use `{{ python_min }}`
```suggestion
    - python {{ python_min }}
```